### PR TITLE
feat(eth): `Polaris` object, `NetworkingStack` modularity, match geth arch in general

### PR DESCRIPTION
**Pull request by itsdevbear**

### CODE REVIEW

**Comment by calbera** [comment]
yeah.. this bangs

File: eth/polar/mining.go [hunk 1]
```diff
@@ -18,26 +18,28 @@
...
-		API{
-			Namespace: "net",
-			Service:   api.NewNetAPI(apiBackend),
-		},
-		API{
-			Namespace: "web3",
-			Service:   api.NewWeb3API(apiBackend),
-		},
-	)
+// TODO: replace this file with a proper mining object and use message passing instead of direct calls.
```
Comment (calbera) [inline eth/polar/mining.go:30]: we need a "ABCI" like messaging protocol between a host chain and the Polaris Eth framework

Comment (itsdevbear) [reply eth/polar/mining.go:30]: basically ye

File: eth/polar/polaris.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+// StartServices notifies the NetworkStack to spin up (i.e json-rpc).
+func (pl *Polaris) StartServices() error {
+	// Register the JSON-RPCs with the networking stack.
+	pl.stack.RegisterAPIs(pl.APIs())
+
+	// Register the filter API separately in order to get access to the filterSystem
+	pl.filterSystem = utils.RegisterFilterAPI(pl.stack, pl.backend, &defaultEthConfig)
+
+	// Register the GraphQL API (todo update cors stuff)
```
Comment (oofncubed) [inline eth/polar/polaris.go:132]: should gate this with a config flag

Comment (oofncubed) [reply eth/polar/polaris.go:132]: or leave TODO

**Pull request merged**